tests/phpunit/tests/test-columns.php

```diff
@@ -257,20 +257,34 @@ public function test_post_inline_edit() {
 		$list_table->inline_edit();
 		$form = ob_get_clean();
 		$doc = new DomDocument();
-		$doc->loadHTML( $form );
+		$doc->loadHTML( $form, LIBXML_NOERROR );
```

Comment:
Instead of ignoring the error, I suggest to open a ticket on wp.org to report the bug introduced by https://github.com/WordPress/WordPress/commit/337e4a8d9075e8376a3e39c8993ddc8e7d953d65

Reply:
Ticket opened [here](https://core.trac.wordpress.org/ticket/55575).

Reply:
Thanks. Would it be a good idea to remove the problematic id from `$form` with some `str_replace()` before passing it to `loadHTML()`? This would avoid to ignore all errors including those we could add ourselves.